packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/ios/initializeflow.md

```diff
@@ -0,0 +1,44 @@
+You can provide the configuration information in a property list file or in the initialization call.
+
+#### Add configuration with a property list
+
+The default `init()` function of `InteractionCodeFlow` looks for a property list called `Okta.plist`. You can specify a property list file by calling `init(plist:)`. Both of those options can result in a throws as they read from the file system. To specify the values in the call, use `init(
```

Comment:
Just checking that "can result in a throws" make sense to a swift user. Should it say "can result in an exception" or an error?

Reply:
Good catch. "an exception" is correct. I'll change it.

Reply:
Fixed in the next commit.